examples/auth/app/auth/pages/login.tsx

```diff
@@ -10,7 +10,7 @@ const LoginPage: BlitzPage = () => {
     <div>
       <LoginForm
         onSuccess={() => {
-          const next = (router.query.next as string) ?? "/"
+          const next = decodeURIComponent(router.query.next as string) ?? "/"
```

Comment:
Hey, thank you! 

The issue is that `decodeURIComponent(undefined) === "undefined"` and so this doesn't fall back to `"/"`.

So probably need to change like this:
```suggestion
          const next = router.query.next ? decodeURIComponent(router.query.next) : "/"
```

Reply:
That makes sense. Updated.